code/modules/jobs/job_types/janitor.dm

```diff
@@ -24,6 +24,7 @@
 	ears = /obj/item/radio/headset/headset_srv
 	uniform = /obj/item/clothing/under/rank/civilian/janitor
 	backpack_contents = list(/obj/item/modular_computer/tablet/preset/advanced=1)
+	skillchips = list(/obj/item/skillchip/job/engineer, /obj/item/skillchip/light_remover)
```

Comment:
unmentioned balance change thats also really powerful 🤔

Reply:
One is removing light tubes without gloves while the other is door wires? Is that right?

Reply:
yes

Reply:
Oh shit, I didn't mean to leave that in, that was for testing :sweat:

Reply:
Light remover makes sense. I don't get the engineer one. The janitor should have a chipset that makes them clean faster.

Reply:
Feel free to make a PR adding the light remover skillchip to the janitor after this one is merged, but this is just supposed to be pure refactoring.